rules/manager/manager.go

```diff
@@ -229,9 +236,11 @@ func (m *ruleManager) runIteration(results chan<- *extraction.Result) {
 					Timestamp: s.Timestamp,
 				}
 			}
-			m.results <- &extraction.Result{
-				Samples: samples,
-				Err:     err,
+
+			if err != nil {
+				evalFailures.Inc()
```

Comment:
Should we log something here too? Knowing what the rule and error were would be handy for fixing things